imp_marl/imp_wrappers/examples/epymarl/README.md

```diff
@@ -0,0 +1,28 @@
+# EPyMARL
+
+Extended PyMARL [EPyMARL](https://github.com/uoe-agents/epymarl) is an extension of [PyMARL](https://github.com/oxwhirl/pymarl), featuring:
+- Additional algorithms (IA2C, IPPO, MAPPO and MAA2C)
+- Option for no-parameter sharing between agents
+- Flexibility with extra implementation details
+
+For more information, check their [blog](https://agents.inf.ed.ac.uk/blog/epymarl/)
+
+You can run IMP-MARL environments on EPyMARL by:
+
+1. Create a new virtual environment.
+2. Install imp_marl package: `pip install imp-marl`.
+3. [Clone EPyMARL](https://github.com/uoe-agents/epymarl).
+4. Install the required EPyMARL modules according to the requirements file: `pip install -r requirements`.
+5. Register your imp_marl env(s) within EPyMARL. Do that on `/src/envs/__init__.py`:
+    ```
+    from imp_marl.imp_wrappers.epymarl_wrapper.epymarl_wrap_ma_struct import EPymarlMAStruct
+    def env_fn(env, **kwargs) -> MultiAgentEnv:
+    return env(**kwargs)
+    REGISTRY = {}
+    REGISTRY["struct_marl"] = partial(env_fn, env=EPymarlMAStruct)
```

Comment:
I think only the first and last lines are not already in the file and one should not overwrite the others?